### DISCUSSION TITLE
Welcome to Flutter-Into-App-Dev Discussions!

### DISCUSSION THREAD

**Discussion by raosush** [ACM-NITK/Flutter-Into-App-Dev#1] [Announcements]
# 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shivaraj-Padala** [comment]
I am using Flutter to write a desktop application and I am using speech to text module in the project but it's giving me this error.
any help is appreciated.
MissingPluginException (MissingPluginException(No implementation found for method initialize on channel plugin.csdcorp.com/speech_to_text))
